### PR TITLE
Init post launch page; Add and style hero

### PR DESCRIPTION
**Pull request by jasondippel**
## What Changed
- Creating post-launch page (currently under /full path as in full-page, open to better ideas)
- Adding and styling hero section of post-launch page

## Screenshots
Kapture 2021-01-22 at 13 01 45

### CODE REVIEW

**Review by julie-huang — DISMISSED** [review]
LGTM but needs to figure out how to deploy 

Don't merge yet please :)

**Comment by jasondippel** [comment]
@julie-huang Thanks for the review! I'm just going to dismiss your approval so I don't accidentally merge this. Will get it re-approved once a plan is in place for how this stuff should be deployed.

**Comment by jasondippel** [comment]
Rather than merging this in, just gonna work off of a `launch` branch 🚀

**Pull request closed without merging**
